cpp/modmesh/buffer/pymod/wrap_SimpleArrayPlex.cpp

```diff
@@ -251,20 +237,7 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapSimpleArrayPlex : public WrapBase<Wr
                         [](auto & array)
                         {
                             using data_type = typename std::remove_reference_t<decltype(array[0])>;
-                            std::vector<size_t> stride;
-                            
-                            for (size_t const i : array.stride())
-                            {
-                                stride.push_back(i * sizeof(data_type));
-                            }
-                            return pybind11::buffer_info(
-                                array.data(), /* Pointer to buffer */
-                                sizeof(data_type), /* Size of one scalar */
-                                pybind11::format_descriptor<data_type>::format(), /* Python struct-style format descriptor */
-                                array.ndim(), /* Number of dimensions */
-                                std::vector<size_t>(array.shape().begin(), array.shape().end()), /* Buffer dimensions */
-                                stride /* Strides (in bytes) for each index */
-                            ); });
+                            return get_buffer_info<data_type>(array); });
```

Comment:
use the function from `array_common.hpp`

Reply:
This (and alike) are good.